src/code/z_demo.c

```diff
@@ -561,9 +575,7 @@ void CutsceneCmd_Destination(PlayState* play, CutsceneContext* csCtx, CsCmdDesti
         titleDemoSkipped = true;
     }
 
-    if ((csCtx->curFrame == cmd->startFrame) || titleDemoSkipped ||
-        ((csCtx->curFrame > 20) && CHECK_BTN_ALL(play->state.input[0].press.button, BTN_START) &&
-         (gSaveContext.fileNum != 0xFEDC))) {
+    if ((csCtx->curFrame == cmd->startFrame) || titleDemoSkipped || DEBUG_SKIP_CS) {
```

Comment:
Why not inline `OOT_DEBUG && ...`, too unwieldy? I checked that it matches, and I don't think it looks terrible when formatted:

```
if ((csCtx->curFrame == cmd->startFrame) || titleDemoSkipped ||
        // In debug mode, cutscenes can be skipped by pressing the start button (or whatever this does)
        (OOT_DEBUG && ((csCtx->curFrame > 20) && CHECK_BTN_ALL(play->state.input[0].press.button, BTN_START) &&
                       (gSaveContext.fileNum != 0xFEDC)))) {
```

Reply:
I never tried it, cause the define didnt exist at the time. I kinda like the complicated debug define being separate in this case, cause of how important the condition is. But I also dont like the condition being defined so far away from its usage...

Not a huge fan of comments in the middle of conditions. its not something we really do at all so far. it may end up looking okay in this case but I can also see it going really badly depending on the length of the different lines.

for now ill go with using the define, without a comment (I already removed them for now per another review above)